src/Shellbar/__stories__/Shellbar.stories.js

```diff
@@ -203,3 +204,136 @@ export const coPilot = () => (
+                title: 'Fiori Home',
+                subtitle: 'Central Home',
+                image: './assets/01.png',
+                selected: true,
+                glyph: 'home'
+            },
+            {
+                callback: () => console.log(`${this.title} selected!`),
+                title: 'S/4 HANA Cloud',
+                image: './assets/02.png',
+                glyph: 'cloud'
+            },
+            {
+                callback: () => console.log(`${this.title} selected!`),
+                title: 'Analytics Cloud',
+                subtitle: 'Analytics Cloud',
+                image: './assets/03.png',
+                glyph: 'business-objects-experience'
+            },
+            {
+                callback: () => console.log(`${this.title} selected!`),
+                title: 'Ariba',
+                image: './assets/04.png',
+                glyph: 'activate'
+            },
+            {
+                callback: () => console.log(`${this.title} selected!`),
+                title: 'SuccessFactors',
+                image: './assets/05.png',
+                glyph: 'message-success'
+            },
+            {
+                callback: () => console.log(`${this.title} selected!`),
+                title: 'Commerce Cloud',
+                image: './assets/06.png',
+                glyph: 'retail-store'
+            },
+            {
+                callback: () => console.log(`${this.title} selected!`),
+                title: 'Gigya',
+                image: './assets/07.png',
+                glyph: 'customer-view'
+            },
+            {
+                callback: () => console.log(`${this.title} selected!`),
+                title: 'Callidus Cloud',
+                image: './assets/08.png',
+                glyph: 'globe'
+            },
+            {
+                callback: () => console.log(`${this.title} selected!`),
+                title: 'Fieldglass',
+                image: './assets/09.png',
+                glyph: 'work-history'
+            },
+            {
+                callback: () => console.log(`${this.title} selected!`),
+                title: 'Concur',
+                image: './10.png',
+                glyph: 'area-chart'
+            },
+            {
+                callback: () => console.log(`${this.title} selected!`),
+                title: 'Cloud for Customer',
+                image: './11.png',
+                glyph: 'customer-view'
+            },
+            {
+                callback: () => console.log(`${this.title} selected!`),
+                title: 'Cloud Portal',
+                image: './12.png',
+                glyph: 'customer'
+            }
+        ]}
+        productTitle='Corporate Portal'
+        profile={profile}
+        profileMenu={profileMenu}
+        searchInput={{
+            label: 'Search',
+            placeholder: 'Search'
+        }}
+        size={select('size', { 's': 's', 'm': 'm', 'l': 'l', 'xl': 'xl' })}
```

Comment:
default to `s`
```suggestion
        size={select('size', { 's': 's', 'm': 'm', 'l': 'l', 'xl': 'xl' }, 's')}
```

Reply:
Good idea! I believe typically 'm' is the default size (at least styling-wise) if you do not pass the `size` prop. With `s` as default at least you will be able to see the difference between the normal co-pilot `Shellbar` and the `s` `Shellbar`.